etc/install.sh

```diff
@@ -96,16 +96,16 @@ fi
 #-----------------------------------------------------------------------------
 # Bootstrap cpanm
 
-CPANM_URL="https://raw.github.com/thaljef/Pinto/master/etc/cpanm"
+PINTO_CPANM_URL=${CPANM_URL:="https://raw.githubusercontent.com/thaljef/Pinto/master/etc/cpanm"}
```

Comment:
Absolutely. Nice catch.


Kahlil (Kal) Hodgson                       GPG: C9A02289
Head of Technology                         (m) +61 (0) 4 2573 0382
DealMax Pty Ltd

Suite 1416
401 Docklands Drive
Docklands VIC 3008 Australia

"All parts should go together without forcing.  You must remember that
the parts you are reassembling were disassembled by you.  Therefore,
if you can't get them together again, there must be a reason.  By all
means, do not use a hammer."  -- IBM maintenance manual, 1925

On 6 February 2015 at 16:40, Jeffrey Ryan Thalhammer <
notifications@github.com> wrote:

> In etc/install.sh
> <https://github.com/thaljef/Pinto/pull/187#discussion_r24223680>:
>
> > @@ -96,16 +96,16 @@ fi
> >  #-----------------------------------------------------------------------------
> >  # Bootstrap cpanm
> >
> > -CPANM_URL="https://raw.github.com/thaljef/Pinto/master/etc/cpanm"
> > +PINTO_CPANM_URL=${CPANM_URL:="https://raw.githubusercontent.com/thaljef/Pinto/master/etc/cpanm"}
>
> Shouldn't these both be called PINTO_CPANM_URL?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thaljef/Pinto/pull/187/files#r24223680>.
>